soc/nxp/mcx/mcxw/soc.c

```diff
@@ -39,6 +39,7 @@ static ALWAYS_INLINE void clock_init(void)
 	};
 	/* Enable OSC32K */
 	CCM32K_Set32kOscConfig(CCM32K, kCCM32K_Enable32kHzCrystalOsc, &ccm32k_osc_config);
+
```

Comment:
no problem with this newline but it seems out of place on the commit, since this file is not touched otherwise, was it intentional?

Reply:
Intentional, however not necessary, only for clarity in code, let me know if you want me to remove this.